packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -50,14 +41,18 @@ const getTemplateValues = (linkType: LinkType) => {
       throw new Error('Invalid link type passed into LinkExpired component');
   }
   return templateValues;
-};
+}
 
-const LinkExpired = ({ linkType }: LinkExpiredProps) => {
+const LinkExpired = ({ linkType, resendLinkHandler }: LinkExpiredProps) => {
   // TODO : Metric event(s) for expired link
 
-  const resendLinkHandler = getResendLinkHandler(linkType);
   const templateValues = getTemplateValues(linkType);
-
+  const onClickReceiveNewLink = () => {
```

Comment:
If we made the `resendLinkHandler` mandatory (and we probably should?), we might not need this error handling?

Reply:
I wanted to make it mandatory, but this would have broken the compilation of other components, and I am trying to keep this PR to minimum since it is already quite big. 

It appears that other components have simply neglected to hook this up. So it felt like an inevitable follow up ticket.

Reply:
I could change this prop to mandatory in this follow up ticket, if that works for you: 
https://mozilla-hub.atlassian.net/browse/FXA-6891

Reply:
That sounds good to me 👍